docs/recipes/cvar-replace.rst

```diff
@@ -0,0 +1,45 @@
+Replacing Contents of a Construction Variable
+---------------------------------------------
+
+The
+`Replace <https://scons.org/doc/production/HTML/scons-man.html#f-Replace>`_
+method of a construction enviromnent replaces the entire existing value.
+It can be convenient to use a Python dictionary and unpacking to set many
+values at once:
+
+.. code-block:: python
+
+   values = {"AFLAGS": "ONE", "BFLAGS": "TWO", "EXTRAFLAGS": ["THREE", "FOUR"]}
+   env.Replace(**values)
+
+The method replaces the entire value for each variable,
```

Comment:
Not really the same. specifying in Program() creates and override environment
more like

`env['BFLAGS']="THREE"`

Reply:
Quite true. I was going to backtrack and introduce the override wording in the previous recipe:

https://scons-cookbook.readthedocs.io/en/latest/#setting-a-compilation-flag-while-calling-a-builder

and in anticipation of that, I changed my original example to call Program...  and then got distracted and didn't complete the loop.  Maybe I should go back to what I had here before.

Reply:
Yup. go back.. ;) 
it's a devil in the details issue, but if we have examples which aren't quite exact we'll likely get more confused users later.